flink-formats/flink-compress/src/test/java/org/apache/flink/formats/compress/CompressionFactoryITCase.java

```diff
@@ -71,8 +72,8 @@ void testWriteCompressedFile(@TempDir java.nio.file.Path tmpDir) throws Exceptio
                 env.addSource(new FiniteTestSource<>(testData), TypeInformation.of(String.class));
 
         stream.map(str -> str)
-                .addSink(
-                        StreamingFileSink.forBulkFormat(
```

Comment:
This should've removed the related import. It didn't because of references in the class javadoc, which also should be updated. also applies to other classes.